### DISCUSSION TITLE
Is mitmproxy now supporting body content filtering?

### DISCUSSION THREAD

**Discussion by kurtxiangli** [mitmproxy/mitmproxy#6071] [General]
Hello,

I wanted to check if now the mitmproxy supports the filters for body content? According to issue "Search or highlight by body contents. #10", since it's opening I guess this feature is not supported yet? If the case, do we have ETA to have this implemented? And is there any other way to inspect and manipulate the content?

Thanks

**Comment by mhils** [comment]
mitmproxy always had body filtering, only mitmweb does not have body filters. No ETA yet unfortunately.

**Comment by kurtxiangli** [comment]
oh, I see. Thanks a lot. The reason why I used mitmweb is that it seems mitmproxy console hangs(not response for pressing any keys) when handling the TLS/DTLS traffic in wireguard mode and mitmweb doesn't have this problem. Let me try different versions and get back here. 


Best,
